modules/Db2/templates/db2.template.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: db2u.databases.ibm.com/v1
```

Comment:
This should be named filename.yaml.tmpl so that it won't ever be interpreted as a yaml until the necessary variables have been replaced.

Reply:
Fixed